lib/league_statistics.rb

```diff
@@ -0,0 +1,111 @@
+module LeagueStatistics
+
+  def count_of_teams
+    teams.count
+  end
+
+  def unique_teams
+    team_id = game_teams.uniq { |game_team| game_team.team_id }
+    team_id.map do |game_team|
+      game_team.team_id
+    end
```

Comment:
This is another option. It should be a tiny tiny bit faster because it uses one less loop but using reduce sacrifices the readability of using uniq and map.

```
game_teams.reduce([]) do |unique_team_ids, game_team|
    id = game_team.team_id
    unique_team_ids << id if !unique_team_ids.include?(id)
    unique_team_ids
end
```